app/models/container_definition.rb

```diff
@@ -12,7 +12,6 @@ def disconnect_inv
     _log.info "Disconnecting Container definition [#{name}] id [#{id}]"
     self.container.try(:disconnect_inv)
     self.deleted_on = Time.now.utc
-    self.container_group = nil
```

Comment:
@simon3z In order to know which project a container belonged to, i need to keep this relationship and use
```has_one :old_container_project, :through :container_group```